client/src/components/ElementEditor/Header.js

```diff
@@ -66,31 +88,43 @@ class Header extends Component {
    * @returns {ActionMenuComponent|null}
    */
   renderActionsMenu() {
-    const { id, expandable, ActionMenuComponent } = this.props;
+    const { id, expandable, ActionMenuComponent, isPublished } = this.props;
 
     // Don't show the menu when inline editing is not enabled
     if (!expandable) {
       return null;
     }
 
     const deleteTitle = i18n._t('ElementHeader.DELETE', 'Delete');
-    const deleteButtonClassNames = classNames('dropdown-item', 'element-editor__actions-delete');
+    const publishTitle = i18n._t('ElementHeader.PUBLISH', 'Publish');
```

Comment:
I'm not convinced these need to be in variables since we aren't re-using them

Reply:
Cool, can easily change that.

Reply:
When you say 'these' do you mean only the `deleteButtonClassNames` or the `publishTitle`/`deleteTitle` also? For the latter I reckoned it be better then calling the translate function twice.

Reply:
The button classes can stay I reckon, but the two titles don't need to be variables if they're only used once. If they are used more than once then they're fine in a variable

Reply:
Used more than once so I'll keep them as they are.